### ISSUE TITLE
Can't find documentation for some resources like `keycloak_openid_client_authorization_resource`

### ISSUE THREAD

**Issue by malacoss** [mrparkers/terraform-provider-keycloak#563]
Hi,

I'm not able to find the online documentation for resources :
- keycloak_openid_client_authorization_resource
- keycloak_openid_client_authorization_permission

I found them in the example from https://registry.terraform.io/providers/mrparkers/keycloak/latest/docs/data-sources/openid_client_authorization_policy

Am I missing something?
Bests

**Comment by diamondq** [comment]
Yes, the documentation is missing for a number of the resources. I just ended up going into the code for the documentation. All the data sources and resources can be found at https://github.com/mrparkers/terraform-provider-keycloak/tree/master/provider. For example,  at

https://github.com/mrparkers/terraform-provider-keycloak/blob/5494ced093114e2e08980dd785827ccad9be59ef/provider/resource_keycloak_openid_client_authorization_permission.go#L25

you'll see the schema requirements for the keycloak_openid_client_authorization_permission.
